src/sagas/trending.js

```diff
@@ -0,0 +1,21 @@
+import {FETCH_TRENDING_GIFS, fetchTrendingGifsFail, fetchTrendingGifsOk} from "../actions/trending";
+import {call, put, takeEvery} from "redux-saga/effects";
+import get from './giphyAPI';
+import {simplifyGIFObject} from "./utils";
+
+function * onFetchTrendingGifs () {
+  let response;
+  try {
+    response = yield call(get, 'trending?limit=80');
+  } catch (error) {
+    yield put(fetchTrendingGifsFail());
+  }
+  const trendingGifs = response.map(simplifyGIFObject);
```

Comment:
These lines `13` and `14` should be inside the try block. Look at this case: if there's an error you are going to dispatch two actions: `fetchTrendingGifsFail` and `fetchTrendingGifsOk`.